interpreters/bas/bas_value.c

```diff
@@ -2026,18 +2042,29 @@ struct String *Value_toWrite(struct Value *this, struct String *s)
 
 struct Value *Value_nullValue(enum ValueType type)
 {
-  static struct Value integer = { V_INTEGER };
-  static struct Value real = { V_REAL };
-  static struct Value string = { V_STRING };
-  static char n[] = "";
+  static struct Value integer =
+  {
+    V_INTEGER
+  };
+
+  static struct Value real =
+  {
+    V_REAL
+  };
+
+  static struct Value string =
+  {
+    V_STRING
+  };
+
   static int init = 0;
```

Comment:
Also I do not see where `init` is set to 1. Is that needed at all? Can we go with static initialization for all the structures?

Reply:
> Also I do not see where `init` is set to 1. Is that needed at all? Can we go with static initialization for all the structures?

No, struct Value contain union, it's hard to initialize union with c89 syntax. So, I add code to set init to 1

Reply:
but the init value point to a string which is an null string, not zero.

Reply:
Yeap. Missed that. So there is no other way.